components/compositing/compositor.rs

```diff
@@ -1856,16 +1856,12 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
         };
 
         // Perform the page flip. This will likely block for a while.
```

Comment:
(if we remove the flag, we should update this comment)

Reply:
I removed `external_present` and updated this comment.